src/commands/showReferrer.ts

```diff
@@ -1,82 +1,46 @@
 import * as vscode from 'vscode';
-import { execAsync } from '../support/execAsync';
+import { execAsync } from '../utils/execAsync';
 import { window } from 'vscode';
-import * as os from 'os';
-import * as path from 'path';
-import { access } from 'fs/promises'; 
+import { logInToDockerCli } from './registries/logInToDockerCli';
+import { checkCli } from '../utils/checksCli';
 
+const cliName = 'oras';
 const goToOrasButton = 'Download ORAS';
 const validTag = /^[a-z0-9./_:-]+$/; 
 const orasURL = 'https://oras.land/docs/installation';
-
-// checks that oras exists on user computer through sending a dummy command. Based on the output exists we return a bool
-async function checkOras(): Promise<boolean> {
-    var errorMessage = `oras executable/binary not user's path environment variable. Download ORAS or update path: `;
-
-        try {
-            var result = await execAsync('oras -h');
-            if (result.stderr){
-                throw new Error;
-            }else{
-            return true ;
-            }
-        } catch {
-            vscode.window.showErrorMessage(errorMessage, goToOrasButton)
-            .then(selection => {
-                if (selection === goToOrasButton) {
-                    vscode.env.openExternal(vscode.Uri.parse(orasURL));
-                } 
-            });
-            return false;
-        } 
-}
-/**
- * an interface that builds the method getDockerCLiCredentials and returns object type any.
- * We need this to mimic the Docker extensions logInToDockerCli code : node.getDockerCliCredentials because we
- * can't get the function itself through the instance of the docker extension.
- */
-interface IAuthentication {
-    getDockerCliCredentials : () => any; 
-}
+const errorMessage = `oras executable/binary not in user's path environment variables. Download ORAS or update path: `;
```

Comment:
backticks` are specifically used for template literals (string replacement kind of things). Use "" or '' for strings